Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -828,6 +830,22 @@
     sprite: Objects/Fun/toys.rsi
     heldPrefix: footb
 
+- type: entity
+  parent: BaseItem
+  id: BeachBall
+  name: beach ball
+  description: Magic ball of solar happiness.
```

Comment:
just use the usual desc `The simple beach ball is one of Nanotrasen's most popular products. 'Why do we make beach balls? Because we can! (TM)' - Nanotrasen`